docs/reports/source-coverage-260519.md

```
@@ -0,0 +1,131 @@
+
+| Module | src .kt | test .kt |
+|---|---:|---:|
+| `core-base/analytics` | 16 | 1 |
+| `core-base/common` | 12 | 0 |
+| `core-base/database` | 5 | 0 |
+| `core-base/datastore` | 10 | 0 |
+| `core-base/designsystem` | 26 | 0 |
+| `core-base/network` | 9 | 0 |
+| `core-base/platform` | 22 | 0 |
+| `core-base/security` | 51 | 7 |
+| **`core-base/store`** | **27** | **26** | ← gold-standard exemplar
```

markdownlint-cli2 (0.22.1)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)